deterministic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.3"
+  spec.add_development_dependency "bundler", '~> 2'
```

Comment:
You're current version basically means `>= 2`. This should be `~> 2.0` which means `>= 2.0` and `< 3.0`

Reply:
see: https://stackoverflow.com/questions/5170547/what-does-tilde-greater-than-mean-in-ruby-gem-dependencies

Reply:
Fixed.